Implementation/FindMyBLEDevice/FindMyBLEDevice/Services/Database/DevicesStore.cs

```diff
@@ -18,7 +18,21 @@ public class DevicesStore : IDevicesStore
 
         private readonly IDatabase _database;
 
-        public BTDevice SelectedDevice { get; set; }
+        private BTDevice _selectedDevice;
+        public BTDevice SelectedDevice
+        {
+            get => new BTDevice
+            {
+                UserLabel = "SomeDevice",
+                LastGPSTimestamp = DateTime.Now,
+                LastGPSLatitude = 49,
+                LastGPSLongitude = 11
+            };
+            set
+            {
+                _selectedDevice = value;
+            }
+        }
```

Comment:
Please revert these changes

Reply:
Nice catch! Totally forgot about the mocked selected device, thanks!